ckan/model/group.py

```diff
@@ -352,12 +357,11 @@ def all_related_revisions(self):
         this group. Ordered by most recent first.
         '''
         results = {}
-        from group_extra import GroupExtra
         for grp_rev in self.all_revisions:
             if not grp_rev.revision in results:
                 results[grp_rev.revision] = []
             results[grp_rev.revision].append(grp_rev)
-        for class_ in [Member, GroupExtra]:
+        for class_ in [Member]:  # GroupExtra is not revisioned any more
```

Comment:
Doesn't need to be a loop any longer, might as well simplify it while we're here.

Reply:
Tis true. However most revision bits, like this method, will be deleted in another PR (along the lines of https://github.com/ckan/ckan/pull/4664/files#diff-63e3d56154b1b0ef0681f88666b149c9L360 ) which should be merged shortly afterwards (in the same release of CKAN), so really this code just needs to not be in a broken state for the meantime.